front_end/ui/legacy/RemoteDebuggingTerminatedScreen.ts

```diff
@@ -27,12 +27,24 @@ const UIStrings = {
    * device back in a state where it can be inspected, before DevTools can reconnect to it.
    */
   reconnectWhenReadyByReopening: 'Reconnect when ready by reopening DevTools.',
+  /**
+   * @description Text in a dialog box to explain `DevTools` can still be used while disconnected.
+   */
+  perserveState: 'Dismiss this dialog and continue using `DevTools` while disconnected.',
```

Comment:
Unnecessary backticks around "DevTools"

Reply:
CDT has those in some existing strings. I suppose it gets swapped for something else if some build flags are specified? Let's do a pass on all of those and see if they fit our needs.